### ISSUE TITLE
Insecure UI validator image breaks green https status

### ISSUE THREAD

**Issue by bravo-kernel** [alt3/cakephp-swagger#5]
Browser complains about insecure media when connecting to UI using HTTPS..

**Update:** 
The warning is a known issue, to be fixed by the swagger team, caused by the Ui-generated validator button so...  no issues in our `index.ctp` as explained by Chrome:

```
Mixed Content: The page at 'https://api.local/swagger' was loaded over HTTPS, but requested an insecure image 'http://online.swagger.io/validator/?url=https://api.local/swagger/docs/api'. This content should also be served over HTTPS.
```

See also:
- https://github.com/swagger-api/swagger-ui/issues/1441
- https://github.com/swagger-api/swagger-ui/pull/1427

**Comment by TeChn4K** [comment]
Same problem here !

**Comment by bravo-kernel** [comment]
Solved in #27 by bumping Swagger UI to 2.2.3
